flink-runtime/src/main/java/org/apache/flink/runtime/util/EnvironmentInformation.java

```diff
@@ -103,6 +103,15 @@ public static String getGitCommitTimeString() {
 		return getVersionsInstance().gitCommitTimeStr;
 	}
 
+	/**
+	 * Returns the exact name of the Dockerimage for Flink that is needed to run in.
+	 *
+	 * @return The "name:tag" of the Flink docker image.
+	 */
+	public static String getFlinkDockerImageNameAndTag() {
```

Comment:
Why  do you think this function belongs to `EnvironmentInformation `?
It looks to me as just a private helper method for `CONTAINER_IMAGE` option for now.

Reply:
Understood. I've moved it to the simplest implementation I can think of (i.e. only in the KubernetesConfigOptions ). If the real usecase appears I expect it to be more clear what the appropriate "central" location should be.